src/CMakeLists.txt

```diff
@@ -40,11 +40,9 @@ add_library(${LIB_NAME} ${BUILD}
         $<TARGET_OBJECTS:sha256>
         $<TARGET_OBJECTS:bigdecimal>
         $<TARGET_OBJECTS:vblake>
+        $<TARGET_OBJECTS:secp256k1>
```

Comment:
Why this change? It will block from linking with BTC.

Reply:
I have changed it because locally with shared build, tests does not link to the seck256 symbols. So I have discussed with Bohdan and we decided to switch to the old variant

Reply:
@blackyblack object libraries, as turned out, can not be linked to libs (source is docs)

Reply:
How did it work on my workstations? Both Linux and Windows linked and worked fine.